src/luminous/__DOCKERFILE_CEPH_VERSION__

```diff
@@ -0,0 +1 @@
+luminous
```

Comment:
Can't we extract that information for the makefile variable ? That would avoid a new file.